pynicotine/gtkgui/frame.py

```diff
@@ -905,14 +909,20 @@ def on_connect(self, *args):
             self.np.servertimer.cancel()
             self.np.servertimer = None
 
+        if self.away:
+            self._apply_away_state(False)
```

Comment:
As far as I can tell, this call is now redundant, and also conflicts with the "in progress" status set earlier in the function (...). The away state is already set in init_interface, called after connecting and logging in.

Reply:
That's true. Fixed in 1f09f60